pkg/server/handler/v1alpha1/webhook.go

```diff
@@ -184,10 +185,14 @@ func createWorkflowRun(tenant string, wft v1alpha1.WorkflowTrigger, data *scm.Ev
 		return err
 	}
 
+	if tag == "" {
```

Comment:
Should add this logic in https://github.com/caicloud/cyclone/pull/1145/files#diff-4db6e7deb63057dec099088e5e228cfbR160.

```
	suffix := rand.String(5)
	name := fmt.Sprintf("%s-%s", wfName, suffix)
	alias := name
	if tag != "" {
		alias = tag
	} else {
		tag = suffix
	}
```